hack/generated/controllers/suite_test.go

```diff
@@ -41,19 +55,101 @@ func (tc *ControllerTestContext) SharedResourceGroupOwner() genruntime.KnownReso
 	return genruntime.KnownResourceReference{Name: tc.SharedResourceGroup.Name}
 }
 
-func setup() error {
+func setupEnvTest() (*rest.Config, error) {
+	envtestContext = &EnvtestContext{
+		testenv: envtest.Environment{
+			CRDDirectoryPaths: []string{
+				"../config/crd/bases/valid",
```

Comment:
TODO: I made this 'valid' directory to put only the needed CRDs in as not all of them are able to be loaded (due to the Any-type stuff that @babbageclunk is working on…)